### PR TITLE
add link interactions

### PR DESCRIPTION
**Pull request by thasipablo**
- Chose "Around The Table", (a tech annual forum for knowledge sharing), as the theme
- Create the 2 pages of the site for mobile (Home and About).
- Adapt the 2 pages to the desktop screen sizes.
- Implement the user interactions: link, mobile menu.
- Implement the section "Featured speakers" with dynamic HTML.
- Deploy the project.

### CODE REVIEW

File: index.html
Comment (Alusp) [inline index.html:1]: ` <header class="header">`

- [ ] Good job so far, :clap: , however, you are suppose to add a top nav for `social icons and links` in your header as part of the project requirement. 

Below is the Screenshot showing the difference: :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
Student Top Nav

</td>
      <td>

Design Top NAv

</td>
    </tr>
  </table>

- [ ] I can see that there are no `Partners section` and `footer in your index.html file`, please fix it to match the given design requirement.

 Below is the screenshot showing the given Design: :arrow_down:

Home P Design Partner and Fotter Sections

File: about.html
Comment (Alusp) [inline about.html:1]: - [ ] Please ensure that your `forum item` overlay matches the design. matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is the Screenshot showing the difference: :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
Student Overlay

</td>
      <td>

Design about Overlay

</td>
    </tr>
  </table>

Comment (Alusp) [inline about.html:1]: - [ ] Please ensure your about section footer matches the design. matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is the Screenshot showing the difference: :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
Student abt footer

</td>
      <td>

design abt footer

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @thasipablo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js [hunk 1]
```diff
@@ -37,60 +37,67 @@ const speakersList = [
   },
 ];
 
-const hamburgerMenuBtn = document.querySelector(".hamburger-menu");
-const closeMobileMenuBtn = document.querySelector(".close-menu");
-const mobileMenu = document.querySelector(".mobile-menu-wrapper");
+const hamburgerMenuBtn = document.querySelector('.hamburger-menu');
+const closeMobileMenuBtn = document.querySelector('.close-menu');
+const mobileMenu = document.querySelector('.mobile-menu-wrapper');
```
Comment (1GORDON) [inline js/index.js:42]: Great design @thasipablo 🤝

- [ ] [Optional] Kindly ensure there is an option to navigate to the home page from the About page of the mobile view.
mobile-about

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @thasipablo ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Professional Readme file ✔️
- Improved UI ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (thasipablo) [reply js/index.js:42]: Thank you for your time reviewing my code. I am going to add it.
Thank you.

**Pull request merged**